common/script/roleInfo.js

```diff
@@ -4914,8 +4902,32 @@
   "none": {
     "name": "none",
     "fragID": "none",
-    "localprops": [],
-    "parentRoles": []
+    "parentRoles": [
```

Comment:
This change is due to none become the default and presentation the alias in #1945